.github/workflows/documentation.yml

```
@@ -0,0 +1,51 @@
+name: Documentation
+
+on: [push]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    permissions:
+    #   contents: write
+      pages: write       # GitHub Pages deployment?
+      id-token: write
+    strategy:
+      max-parallel: 1
+    steps:
+      - uses: actions/checkout@v4
+      - name: Set Python environment
+        uses: actions/setup-python@v5
+        with:
+          python-version: 3.11
+          architecture: x64
+      - name: Install dependencies
+        run: |
+          python -m pip install --upgrade pip
+          python -m pip install uv
+          python -m uv pip install -e .[docs]
+
+      - name: Build static files
+        id: build
+        run: |
+         sphinx-build -M html docs docs/_build
+  
+      - name: Upload static files as artifact
+        id: deployment
+        uses: actions/upload-pages-artifact@v3 # or specific "vX.X.X" version tag for this action
+        with:
+          path: docs/_build
```

```suggestion
          path: docs/_build/html
```
Got an extra directory in the artefact. You can double check this by downloading the artefact, it contains a `artefact.tar` which should have a `index.html` in it, instead there are two directories `doc-tree` and `html`, `html` has the `index.html`

---

Also:
- https://outerl.github.io/jaxlogit/ : 404 (should be the test page)
- https://outerl.github.io/jaxlogit/html : The test page

---

Made this change. Surprisingly the page [https://outerl.github.io/jaxlogit/html](https://outerl.github.io/jaxlogit/html) still works

---

I got a 404 now, perhaps it's just local browser caching.